pom.xml

```diff
@@ -44,4 +44,12 @@
         </repository>
     </repositories>
 
+    <dependencies>
+        <dependency>
+            <groupId>co.nstant.in</groupId>
```

Comment:
this dependency is need in openam or we need to add a maven build step to build the jar with dependencies, minus the parent dependencies.